db/seeds.rb

```diff
@@ -22,13 +37,46 @@
   timezone: 'Central Time (US & Canada)'
 )
 
-user.confirm
+@user_kate.confirm
+puts_records_in_db(User)
+
+# ---------------------------------------------
+# Children
+# ---------------------------------------------
+
+
+# find_or_create_by! a Child with the full_name,
+#  and birthday set randomly between the min_age and max_age.
+def child_named(full_name, min_birthday: (Time.zone.now - 2.weeks),
```

Comment:
Can we extract these min_birthday and max_birthday into constants above?  These might come in handy later.